src/app/admin/modals/create-unit-modal/create-unit-modal.service.ts

```diff
@@ -0,0 +1,23 @@
+// create-unit-modal service
+// Migrated by Mitchell Burcheri
+// Help from Ray Guo
+// Migrated Semester 1 2022
+
+import { Injectable } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { CreateUnitModalComponent } from './create-unit-modal.component';
+
+@Injectable({
+  // creates a provider for the service
+  providedIn: 'root',
+})
+export class CreateUnitModalService {
+  constructor(public dialog: MatDialog) {}
+  show(units): void {
+    // show dialog box for CreateUnitModalComponent
+    const dialogRef = this.dialog.open(CreateUnitModalComponent, {
```

Comment:
You can remove `const dialogRef =` since it doesn't appear to be used by anything

Reply:
Thanks I've just removed it for the next upload